src/calculateRentalCost.js

```diff
@@ -5,6 +5,21 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const payPerDay = 40;
```

Comment:
Also for constants that we use  for explanation values a different syntax will be used, where all letters are capitalized, and instead of a space - an underscore `_` .